### ISSUE TITLE
💡 [REQUEST] - Add model.eval() after loading the mdoel

### ISSUE THREAD

**Issue by Justee21** [pytorch/tutorials#3965]
### 🚀 Describe the improvement or the new tutorial

On this site: https://docs.pytorch.org/tutorials/beginner/basics/saveloadrun_tutorial.html after the last final example and line of code: model = torch.load('model.pth', weights_only=False), it may be helpful to add model.eval() below for clarity and consistency as done earlier in this web page. This makes the recommended inference pattern more explicit and is especially helpful for models that include Batch Normalization or Dropouts.
